protobuf/message.proto

```diff
@@ -3,62 +3,98 @@ package message;
 
 option go_package = "github.com/Zamerykanizowana/replicated-file-system/protobuf";
 
+// Message is a container for either a request or a response from the peer `peer_name` and is related to the transaction denoted by `tid`.
 message Message {
-  // tid is transaction id.
+  // The transaction ID denoted by a Version 4 UUID string.
   string tid = 1;
+  // A textual name of the peer name, e.g. Aragorn.
   string peer_name = 2;
   oneof type {
+    // A request for the pending file-system operation.
     Request request = 3;
+    // A response containing an `ACK` or `NACK`.
     Response response = 4;
   }
 }
 
+// Request provides an almost 1:1 mapping to a syscall-backed file-system operation.
 message Request {
+  // An operation on a file, e.g. create, remove, write, etc.
   Type type = 1;
+  // System call parameters, e.g. path, mode, offset.
   Metadata metadata = 2;
+  // A logical timestamp used for resolving conflicts.
   uint64 clock = 3;
+  // This field is only used for `CREATE` and `WRITE` operations.
   optional bytes content = 4;
 
+  // An operation on a file mapping to a syscall.
   enum Type {
+    // `int creat(const char *pathname, mode_t mode);`
```

Comment:
maybe format these as C code blocks instead?

Reply:
I looked at the generated MD and it might be a bad idea after all, right? :D

Reply:
It would be cool but unfortunately you can't embed code blocks in table rows (not without [HTML tags](https://stackoverflow.com/a/52620288) at least, which are not supported in the LaTeX Markdown thingy).